$MirOS: ports/emulators/qemu/patches/patch-hw_sun4m_c,v 1.3 2005/11/15 12:25:09 tg Exp $
--- hw/sun4m.c.orig	Sun Sep  4 17:11:09 2005
+++ hw/sun4m.c	Tue Nov 15 12:09:56 2005
@@ -58,6 +58,8 @@
 #define PHYS_JJ_FLOPPY_IRQ 22
 #define PHYS_JJ_ME_IRQ 30		/* Module error, power fail */
 
+int load_kernel(const char *, uint8_t *);
+
 /* TSC handling */
 
 uint64_t cpu_get_tsc()
@@ -298,4 +300,5 @@ QEMUMachine sun4m_machine = {
     "sun4m",
     "Sun4m platform",
     sun4m_init,
+    NULL
 };
